conf/params.distrib.xml

```diff
@@ -4,7 +4,9 @@
   <itop_url>https://localhost/iTop</itop_url>
   <itop_login>admin</itop_login>
   <itop_password>admin</itop_password>
-  
+  <rest-token/>
+  <token/>
```

Comment:
Should maybe better be renamed to `itop_token` to be streamlined with other options and some data-collectors might already have `token` in use.

Reply:
good point i will change

Reply:
Seems like this part has not been reworked.

**Point 1**
That being said having both "token" and "rest-token" is really confusing for people not knowing about the first beta version of the `authent-token` module.

Do you have a real use case where the legacy form would already be deployed?

Otherwise, as this (authent-token) has not really be published yet and is not spread out, I would rather not support the old "token" at all as it bring more confusion and technical debt than benefits.
The few people (only 1 customer is **maybe** using it in a CEX integration; and maybe some people using the `combodo-intercom-integration`) will be forced to upgrade this module but it is a totally understandable tradeoff IMO. Just needs to be documented along with all the new changes of the data collector.

**Point 2**
I have to dig into the new version of `authent-token`, but is `rest-token` the name of the param in which lies the token? Why not use something like `authent_[rest_]token` to be consistent with the other parameters. It would also make it explicit that is it for authentication.

Reply:
in next version of authent-token we decided to rename legacy login_mode 'rest-token' to 'token'. for backward compatibility we supported both. let's discuss this apart...